homeassistant/components/squeezebox/media_player.py

```diff
@@ -156,7 +170,17 @@ async def _player_discovered(player: SqueezeBoxPlayerUpdateCoordinator) -> None:
         },
         "async_call_query",
     )
-
+    platform.async_register_entity_service(
+        SERVICE_CALL_QUERY_RESPONSE,
+        {
+            vol.Required(ATTR_COMMAND): cv.string,
+            vol.Optional(ATTR_PARAMETERS): vol.All(
+                cv.ensure_list, vol.Length(min=1), [cv.string]
+            ),
```

Comment:
We generally don't want generic methods like these. They are too technical and error prone. If there is interesting things we want to expose, we should do so in a specific, single-purpose action.

../Frenck